src/MerchantCenter/MerchantStatuses.php

```diff
@@ -137,7 +139,12 @@ public function get_product_statistics( bool $force_refresh = false ): array {
 	 */
 	public function get_issues( string $type = null, int $per_page = 0, int $page = 1, bool $force_refresh = false ): array {
 		$this->maybe_refresh_status_data( $force_refresh );
-		return $this->fetch_issues( $type, $per_page, $page );
+
+		// Get only critical issues
```

Comment:
Minor: We can adjust this comment to just mention error issues

Reply:
Thanks 🙏 Updated here https://github.com/woocommerce/google-listings-and-ads/pull/1437/commits/71823072f6fac6cd65562d21f2d5acfb389aab7f  (still thinking about how I didn't see that one 🤔  tho)